frontend/src/app/testQueue/AoEForm/SymptomInputs.tsx

```diff
@@ -14,7 +17,7 @@ interface Props {
   currentSymptoms: Symptoms;
   setSymptoms: (symptoms: Symptoms) => void;
   onsetDate: string | undefined;
-  setOnsetDate: (onsetDate: string) => void;
+  setOnsetDate: (onsetDate: FormattedDate) => void;
```

Comment:
```suggestion
  setOnsetDate: (onsetDate: ISODate | null) => void;
```
I think this is missing a `null` option as onset date is not a required field

Reply:
Done in 7026199. Along similar lines, is there any reason for optional date values to _not_ allow both `null` _and_ `undefined`?